rust/Cargo.toml.in

```diff
@@ -80,7 +80,7 @@ ldap-parser = { version = "~0.5.0" }
 hex = "~0.4.3"
 psl = "2"
 
-time = "~0.3.41"
+time = "=0.3.41"
```

Comment:
@jasonish question: if patch releases can lead to MSRV bumps for crates, why don't we pin all of the crates?
I had thought we use tilde versioning with major, minimum and patch specified to get security updates for a stable version but this shows there can be more than that.